src/vaults.lua

```diff
@@ -53,8 +55,18 @@ end
 --- @param currentTimestamp number The current timestamp
 --- @param vaultId string The vault id
 --- @param allowUnsafeAddresses boolean|nil Whether to allow unsafe addresses, since this results in funds eventually being sent to an invalid address
---- @return Vault The created vault
-function vaults.vaultedTransfer(from, recipient, qty, lockLengthMs, currentTimestamp, vaultId, allowUnsafeAddresses)
+--- @param revokable boolean|nil Whether the vault is revokable
+--- @return Vault -- The created vault
```

Comment:
Same nit about the return type comments

Reply:
they result in the same output. I'd update them all across the codebase but `--` is a hard grep to search and replace on